en/drafts/originals/space-place-gazetteers.md

```diff
@@ -25,122 +25,111 @@ doi: XX.XXXXX/phen0000
+Names and attestations that are grouped into a single entity are easiest to find and use, but the decision to group disparate pieces of information may come at the expense of precision, accuracy and nuance. The author or team working on a gazetteer may therefore choose to articulate certain disambiguation principles to allow better interoperability and reusability of the data. Beyond human judgement, these questions are the domain of entity resolution, which is an open and unresolved topic in information science, natural language processing, and geoscience.[^9] Spatial historians, as well as information scientists interested in questions of temporality, have also begun to publish on this topic.[^10]
 
-## Building a Gazetteer from a Historical Text
-
-Historians often work with detailed written texts, such as memoirs or travelogues, that may contain a wealth of spatial information. *The Itinerary of Benjamin of Tudela* is one such example of a rich, descriptive historical text that can be data mined for spatial research.
-
-Benjamin of Tudela was a twelfth century Spanish Jewish traveler whose text describes his expedition and his interactions with different Jewish communities. A spatial historian interested in this text may want to discover where Benjamin of Tudela traveled on his grand journey, and how he interacted with Jewish communities in the locations he visited. 
+To be sure, it is a matter of your personal and scholarly judgement, and of your research strategy, to decide whether these names do indeed refer to a single place. After all, Manahatta was the name of an island, not an inhabited place, and that island today is the site of only one of the five boroughs of New York City. There is no objective way to decide whether to group these names together as references to a single place. In an ambiguous case like this, whether you group these names together or not is determined primarily by whether it would enhance your research inquiry and visualization tasks to do so. Alternatively, you could keep these names separate, but still define them as 'relations' using a [Linked Places format](https://github.com/LinkedPasts/linked-places-format), or another similar data format. 
 
-A scholar might also use this source as one element in a larger corpus of texts to examine questions about travel in the post-classical period, European exploration, or Eurasian Jewish studies. The places named in this itinerary could be cross-referenced with those named in other accounts from a similar period to see if there were certain stops that were more popular than others, or to see if different travelers described the locations in the same ways.
+### LP and LP-TSV Formats
```

Comment:
I moved the discussion of this format back up here to follow directly from the first mention right above. This way, we only need to explain it once, and we can easily refer back to it later on.

Reply:
Okay.